string Class
float32 probability
int32 xmin
int32 ymin
int32 xmax
int32 ymax
